support-frontend/assets/pages/supporter-plus-landing/supporterPlusRouter.tsx

```diff
@@ -94,48 +94,46 @@ export const inThreeTier = threeTierCheckoutEnabled(
 	commonState.amounts,
 );
 
+// Lazy load your components
+// const ThreeTierLanding = lazy(() => import('./twoStepPages/threeTierLanding'));
+const SupporterPlusCheckout = lazy(
+	() => import('./twoStepPages/secondStepCheckout'),
+);
+const SupporterPlusThankYou = lazy(
+	() => import('pages/supporter-plus-thank-you/supporterPlusThankYou'),
+);
+
+console.log('***');
+
 // ----- Render ----- //
 
 const router = () => {
 	return (
 		<BrowserRouter>
 			<ScrollToTop />
 			<Provider store={store}>
-				<Routes>
-					{countryIds.map((countryId) => (
-						<>
-							<Route
-								path={`/${countryId}/contribute/:campaignCode?`}
-								element={
-									/*
-									 * if you are coming to the /contribute route(s) and you also have a one off
-									 * contribution type (set in the url) and find yourself in the three tier
-									 * variant we should redirect you to the /contribute/checkout route
-									 */
-									inThreeTier ? (
```

Comment:
Presume need to bring this back if its more than a prototype?

Reply:
Yeah I stripped this out for the prototype as it wasn't necessary to test my hypothesis